### ISSUE TITLE
Softmax op has different result from keras.layers.Softmax and numpy when axis != -1

### ISSUE THREAD

**Issue by jiafatom** [microsoft/onnxruntime#4288]
**Describe the bug**
Consider a simple model here:
```python
    model = keras.Sequential()
    model.add(keras.layers.InputLayer((2, 2, 2)))
    model.add(keras.layers.Softmax(axis=1))
    data = np.random.rand(1, 2, 2, 2).astype(np.float32)
    from scipy.special import softmax
    data_numpy = softmax(data, axis=1)
    onnx_model = keras2onnx.convert_keras(model, model.name)
```
the keras Softmax result is consistent with scipy (a numpy wrapper). But the result is different from onnx Softmax. See onnx model here:
<img width="225" alt="softmax" src="https://user-images.githubusercontent.com/30608893/85187302-7c8f0980-b253-11ea-87ae-58f744f66c85.png">

Only axis=-1 can get the same result. 

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Any
- ONNX Runtime installed from (source or binary): pypi
- ONNX Runtime version: 1.3.0
- Python version: 3.7

**Comment by hariharans29** [comment]
I ll take a look - are you using the CPU package or GPU package ?

**Comment by jiafatom** [comment]
I checked the onnx spec again, and find that the difference is expected -- onnx Softmax can group several axis but keras.layers.Softmax and numpy can only work on one axis.

**Comment by leimao** [comment]
> I ll take a look - are you using the CPU package or GPU package ?

@jiafatom @hariharans29 I think this is a Keras2ONNX bug on Softmax export. The ONNX model inference result should match TF-KERAS inference result, no matter what.
